.github/workflows/integration-test-build-publish.yaml

```diff
@@ -0,0 +1,103 @@
+name: Build and Publish Integration Test Docker Image
+
+on:
+  workflow_dispatch: {}
+  push:
+    branches:
+      - main
+    paths:
+      - Dockerfile.integration-test
+    tags:
+      - v*.*.*
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: greenhouse-extensions-integration-test
+
+jobs:
+  build-and-push:
+    name: Build and push Docker image
+    runs-on: ubuntu-latest
```

Comment:
Please use the GitHub organization provided runner: `runs-on: [ default ]`

Reply:
thank you. I have fixed in https://github.com/cloudoperators/greenhouse-extensions/pull/237/commits/924cf43397727fe509b9c81e936192123c6d484f